azkaban-common/src/main/java/azkaban/executor/ExecutableFlow.java

```diff
@@ -191,6 +202,10 @@ public void setSubmitTime(long submitTime) {
     this.submitTime = submitTime;
   }
 
+  public void setSlaOptions(List<SlaOption> slaOptions) {
+    this.slaOptions = slaOptions;
+  }
+
   @Override
   public Map<String, Object> toObject() {
```

Comment:
agree, submited an issue request for tracking https://github.com/azkaban/azkaban/issues/1098